010-AzureDevOps/Coach/Solution-03.md

```diff
@@ -4,17 +4,6 @@
 
 ## Notes & Guidance
 
-This is the only section you need to include.
-
-Use general non-bulleted text for the beginning of a solution area for this challenge
-
-- Then move into bullets
-  - And sub-bullets and even
-    - sub-sub-bullets
-
-Break things apart with more than one bullet list
-
-- Like this
-- One
-- Right
-- Here
+Download the resources content  in the **(../student/resources)** and zip the content and provide to the students. 
+ - Students can have issues if many of them upload the content in the same time in the same repo.
+ - Recommend to just one execute this challenge and share the screen with others. 
```

Comment:
Agree with the issue where multiple students pushing at the same time will cause issues.  Would recommend revising the recommendation to have each student upload to their own instance of ADO otherwise only 1 student may be doing the work going forward.